packages/core/src/docdb/commands/stopCluster.ts

```diff
@@ -0,0 +1,23 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as vscode from 'vscode'
+import { getLogger } from '../../shared/logger'
+import { telemetry } from '../../shared/telemetry'
+import { localize } from '../../shared/utilities/vsCodeUtils'
+import { DBClusterNode } from '../explorer/dbClusterNode'
+
+export function stopCluster(node?: DBClusterNode): Promise<void> {
+    return telemetry.docdb_stopCluster.run(async () => {
+        if (node?.arn && node?.regionCode) {
+            await node.client.stopCluster(node.arn)
+            getLogger().info('docdb:Stop cluster: %O', node.name)
+            void vscode.window.showInformationMessage(
```

Comment:
Question: Should we display infor before waiting for node.client.stopCluster(node.arn) ?

Reply:
It returns immediately after, doesn't block while stopping